website/docs/docs/build/unit-tests.md

```diff
@@ -26,8 +26,8 @@ With dbt Core v1.8 and dbt Cloud environments that opt to "Keep on latest versio
 - You must specify all fields in a BigQuery STRUCT in a unit test. You cannot use only a subset of fields in a STRUCT.
 - If your model has multiple versions, by default the unit test will run on *all* versions of your model. Read [unit testing versioned models](#unit-testing-versioned-models) for more information.
 - Unit tests must be defined in a YML file in your `models/` directory.
-- Available to dbt Cloud customers who have selected ["Keep on latest version"](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version) and  dbt Core v1.8.0 or later.
```

Comment:
Yeah, this shouldn't have been there. There was previously the exact same note at the top of this list, so apparently it was there twice. We removed it in favor of the banner because folks were missing it